### ISSUE TITLE
refreshItems not working on live site

### ISSUE THREAD

**Issue by mmikaela** [haltu/muuri#224]
Hi, I wrote something very similar to this about a year ago (#111), but the previously resolved issue has evolved lol.

`grid.refreshItems().layout();` works perfectly on my text editor's preview on Chrome; however, when it's hosted somewhere, there's a very weird problem.

On a second load of the page, the grid collapses again. However, the first time you load the page, it's perfect. Basically, any additional page load after the first time(including refreshing), it breaks in one of two ways mentioned.

I don't understand why this happens, because I thought the refreshItems() function works its magic on load, or that `layoutOnInit:true` should be enough. I've tried writing out `layoutOnInit:true` again just in case. I've tried calling `grid.refreshItems().layout();` immediately after my preloader disappears if that was an issue. I've tried putting the initialization script (e.g. "`var grid`") inside the listener `$(document).ready`, `window.addEventListener('load', function())`, and `window.bind('beforeunload', function())`. I've used all three in one script tag, just one at a time, or no listener at all, and all of them have this problem exclusively on a live, hosted site. 

I stopped letting my hosting service cache my website at all and the problem still persists. And yes, I've been testing this in Incognito mode, while making sure I use a whole new window after uploading a change. I'm just really frustrated and don't know what to do.

Here's a "paraphrased" version of my code with the no-listeners attempt:

```
<div class="first">
	<div class="second">
		<div class="grid">
			<div class="item col-md-4 col-xs-12"></div>
  		</div>
 	</div>
</div>
<!-- after html -->
<script>
	 var grid = new Muuri('.grid', {
		items: '.item',
  		dragEnabled: false,
		layout: {
      			fillGaps: true,
	  		rounding: false
		},
		layoutOnInit: true
	});
	grid.refreshItems().layout();
</script>
```
I hope no one judges me but I'll also go ahead and link [the live page](https://mmikaela.com/browse.html) so people can see what I mean or if you need to look at all the other plugins i'm using lol :,) thank you again for quick responses!

**Comment by mmikaela** [comment]
Thanks for the quick response @niklasramo !
**I had to [move the .html file to this URL now](https://mmikaela.com/browse-problem.html), due to my project needing to be updated**. 

I'm sad to say I actually ended up switching from Muuri to Isotope for my project (at least temporarily), which has a pretty similar function. I ran into the same problem with Isotope when I turned off scrolling/`overflow:scroll;` (I think `overflow:scroll;` is necessary to initialize even if the user doesn't scroll). Calling Isotope's version of Muuri's `grid.refreshItems().layout()` after the pre-loader disappears (as you suggested) did help in this case!

I realized about a day after I posted this that the lazy loading isn't actually working in my Muuri example because none of my images use `img data-src=` which is required for it to work (lol). Because my lazy loading wasn't doing anything, the images in my example _should_ be loading before `grid.layout()`. Just in case, i went ahead and deleted the lazy script.

I had tried calling `grid.refreshItems().layout()` after the pre-loader disappears before I posted here, and it didn't work. I tried again today after knowing that this works with Isotope, and it still isn't working. I'm wondering if because `grid.refreshItems().layout()` is in a different script tag than my Muuri initialization with a different event/function that it may not be recognized? Or maybe it's an error?

I really do like Muuri and wish I could keep it for my project, I just needed to get this fixed ASAP.
